core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -641,6 +646,33 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
     }
   }
 
+  def requestAccessToManagedGroup(groupRef: ManagedGroupRef): Future[PerRequestMessage] = {
+    dataSource.inTransaction { dataAccess =>
+      dataAccess.managedGroupQuery.load(groupRef).flatMap {
```

Comment:
Minor nitpick: you could probably remove this outer `flatMap` with a `map` like so:
```
dataAccess.managedGroupQuery.load(groupRef).map {
  case None => // throw exception
  case Some(managedGroup) => {
    dataAccess.rawlsGroupQuery.flattenGroupMembership(managedGroup.adminsGroup).map { users =>
      users.foreach { user =>
        notificationDAO.fireAndForgetNotification(GroupAccessRequestNotification(user.userSubjectId.value, groupRef.membersGroupName.value, managedGroup.adminsGroup.groupEmail.value, userInfo.userEmail))
      }
    }
    RequestComplete(StatusCodes.NoContent)
  }
}
```
Not a big deal but IMO makes it a little more clear that the call to `flattenGroupMembership` doesn't influence the return value.